src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example2.java

```diff
@@ -0,0 +1,11 @@
+/*xml
+<module name="Checker">
+  <module name="NewlineAtEndOfFile"/>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.newlineatendoffile;
+
+public class Example2 { // ⤶
+// ⤶
+} // no ⤶ below
```

Comment:
>Violation lines for /home/vsts/work/1/s/src/xdocs-examples/resources/
com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example2.java differ.
expected to be empty but was: [1]

you do not have comment "// violation" in file but file does have violation during execution.
we  put comment `// violation` to make it visible to user where is violation.

as violation on first line you can do:
```
/*xml
<module name="Checker">
  <module name="NewlineAtEndOfFile"/>
</module>
*/
// violation 5 lines above `.....put here message..`
package com.puppycrawl.tools.checkstyle.checks.newlineatendoffile;
```

Reply:
> > Violation lines for /home/vsts/work/1/s/src/xdocs-examples/resources/
> > com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example2.java differ.
> > expected to be empty but was: [1]
> 
> you do not have comment "// violation" in file but file does have violation during execution. we put comment `// violation` to make it visible to user where is violation.
> 
> as violation on first line you can do:
> 
> ```
> /*xml
> <module name="Checker">
>   <module name="NewlineAtEndOfFile"/>
> </module>
> */
> // violation 5 lines above `.....put here message..`
> package com.puppycrawl.tools.checkstyle.checks.newlineatendoffile;
> ```

still giving the same error

Reply:
all the test cases are passing now, I have pushed the changes but there is another problem, `mvn clean verify` is throwing some errors:-

```
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\xdocs\checks\misc\newlineatendoffile.xml:106: Content of source tag should not be Indented [noIndentationConfigExamples]
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\xdocs\checks\misc\newlineatendoffile.xml:110: Content of source tag should not be Indented [noIndentationConfigExamples]    
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\xdocs\checks\misc\newlineatendoffile.xml:125: Content of source tag should not be Indented [noIndentationConfigExamples]    
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\xdocs\checks\misc\newlineatendoffile.xml:140: Content of source tag should not be Indented [noIndentationConfigExamples]
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\xdocs\checks\misc\newlineatendoffile.xml:144: Content of source tag should not be Indented [noIndentationConfigExamples]  
```

Idk why these errors are occurring, the content of the `newlineatendoffile.xml` is correct I think. And also sometimes after running the `mvn clean verify` the example code in the `<source>` tag gets deleted automatically. Sorry if this PR is taking a long time to merge ;(

Reply:
the file looks likes this when the code gets deleted automatically

```
      <subsection name="Examples" id="Examples">
        <p id="Example1-config">
          To configure the check:
        </p>
        <source>
&lt;module name=&quot;Checker&quot;&gt;
  &lt;module name=&quot;NewlineAtEndOfFile&quot;/&gt;
&lt;/module&gt;
        </source>
        <p id="Example1-code">Example:</p>
        <source>

        </source>
        <p id="Example2-code">Violation Example:</p>
        <source>

        </source>

        <p id="Example3-config">
          To configure the check to always use Unix-style line separators:
        </p>
        <source>
&lt;module name=&quot;Checker&quot;&gt;
  &lt;module name=&quot;NewlineAtEndOfFile&quot;&gt;
    &lt;property name=&quot;lineSeparator&quot; value=&quot;lf&quot;/&gt;
  &lt;/module&gt;
&lt;/module&gt;
        </source>
        <p id="Example3-code">Example:</p>
        <source>

        </source>

        <p id="Example4-config">
          To configure the check to work only on Java, XML and Python files:
        </p>
        <source>
&lt;module name=&quot;Checker&quot;&gt;
  &lt;module name=&quot;NewlineAtEndOfFile&quot;&gt;
    &lt;property name=&quot;fileExtensions&quot; value=&quot;java, xml, py&quot;/&gt;
  &lt;/module&gt;
&lt;/module&gt;
        </source>
        <p id="Example4-code">Python file Example:</p>
        <source>

        </source>
        <p id="Example5-code">Text file Example:</p>
        <source>

        </source>
      </subsection>
```